lib/ansible/modules/cloud/amazon/ec2_group.py

```diff
@@ -315,10 +318,21 @@ def add_rules_to_lookup(ipPermissions, group_id, prefix, dict):
             dict[make_rule_key(prefix, rule, group_id, ipv6Grants.get('CidrIpv6'))] = (rule, ipv6Grants)
 
 
+def rule_description_needs_update(module, rule_id, groupRules, rule):
+    minimum_botocore_version = '1.7.2'
+    if StrictVersion(botocore.__version__) < StrictVersion(minimum_botocore_version):
```

Comment:
Prefer to use something like `hasattr(client, "update_security_group_rule_descriptions_ingress")` to test whether we have a new enough botocore.